Automattic/wp-calypso
# Plans Overhaul: Remove visits restriction and update storage to 1GB for Free plan

#### Changes proposed in this Pull Request

* Remove visit restriction across Free & Pro in plan comparison
* Update storage to 1GB (from 500MB) for Free plan 

Discussion in Slack: p1649150277370239-slack-GSH87270U

#### Media

<img width="928" alt="Screenshot 2022-04-05 at 1 06 08 PM" src="https://user-images.githubusercontent.com/1705499/161738087-0cd7a789-075a-431d-825b-d30ae157e5cd.png">


#### Testing instructions



1. Go to `/start/plans`
2. Ensure there is no "visits" comparison in the features list
3. Ensure storage reads `1GB` for the Free plan (also in the tooltip), as in the media above.
4. Go to `/plans/[SITE_URL]` with a personal site and ensure the same 2 & 3 above.
